<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitReportReporter -->
<testsuite hostname="DESKTOP-CI74CGQ" failures="2" tests="13" name="com.qa.crm.tests.HomePageTest" time="13.104" errors="0" timestamp="2025-03-28T16:31:44 IST" skipped="0">
  <testcase classname="com.qa.crm.tests.HomePageTest" name="homePageTitleTest" time="0.053"/>
  <system-out/>
  <testcase classname="com.qa.crm.tests.HomePageTest" name="isHomePageIconTest" time="2.602"/>
  <system-out/>
  <testcase classname="com.qa.crm.tests.HomePageTest" name="isCalendarPageIconTest" time="0.175"/>
  <system-out/>
  <testcase classname="com.qa.crm.tests.HomePageTest" name="isContactsPageIconTest" time="0.103"/>
  <system-out/>
  <testcase classname="com.qa.crm.tests.HomePageTest" name="homePageHeadersCountTest" time="0.091">
    <failure message="expected [5] but found [3]" type="java.lang.AssertionError">
      <![CDATA[java.lang.AssertionError: expected [5] but found [3]
at org.testng.Assert.fail(Assert.java:99)
at org.testng.Assert.failNotEquals(Assert.java:1037)
at org.testng.Assert.assertEqualsImpl(Assert.java:140)
at org.testng.Assert.assertEquals(Assert.java:122)
at org.testng.Assert.assertEquals(Assert.java:907)
at org.testng.Assert.assertEquals(Assert.java:917)
at com.qa.crm.tests.HomePageTest.homePageHeadersCountTest(HomePageTest.java:49)
at java.base/jdk.internal.reflect.DirectMethodHandleAccessor.invoke(DirectMethodHandleAccessor.java:103)
at java.base/java.lang.reflect.Method.invoke(Method.java:580)
at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:133)
at org.testng.internal.TestInvoker.invokeMethod(TestInvoker.java:598)
at org.testng.internal.TestInvoker.invokeTestMethod(TestInvoker.java:173)
at org.testng.internal.MethodRunner.runInSequence(MethodRunner.java:46)
at org.testng.internal.TestInvoker$MethodInvocationAgent.invoke(TestInvoker.java:824)
at org.testng.internal.TestInvoker.invokeTestMethods(TestInvoker.java:146)
at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:146)
at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:128)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1596)
at org.testng.TestRunner.privateRun(TestRunner.java:794)
at org.testng.TestRunner.run(TestRunner.java:596)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:377)
at org.testng.SuiteRunner.access$000(SuiteRunner.java:28)
at org.testng.SuiteRunner$SuiteWorker.run(SuiteRunner.java:418)
at org.testng.internal.thread.ThreadUtil.lambda$execute$0(ThreadUtil.java:64)
at java.base/java.util.concurrent.FutureTask.run(FutureTask.java:317)
at java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1144)
at java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:642)
at java.base/java.lang.Thread.run(Thread.java:1583)
]]>
    </failure>
  </testcase> <!-- homePageHeadersCountTest -->
  <system-out/>
  <testcase classname="com.qa.crm.tests.HomePageTest" name="homePageHeadersTest" time="0.187">
    <failure message="lists don&#039;t have the same size expected [5] but found [3]" type="java.lang.AssertionError">
      <![CDATA[java.lang.AssertionError: lists don't have the same size expected [5] but found [3]
at org.testng.Assert.fail(Assert.java:99)
at org.testng.Assert.failNotEquals(Assert.java:1037)
at org.testng.Assert.assertEqualsImpl(Assert.java:140)
at org.testng.Assert.assertEquals(Assert.java:122)
at org.testng.Assert.assertEquals(Assert.java:907)
at org.testng.Assert.assertEquals(Assert.java:1089)
at org.testng.Assert.assertEquals(Assert.java:1065)
at com.qa.crm.tests.HomePageTest.homePageHeadersTest(HomePageTest.java:56)
at java.base/jdk.internal.reflect.DirectMethodHandleAccessor.invoke(DirectMethodHandleAccessor.java:103)
at java.base/java.lang.reflect.Method.invoke(Method.java:580)
at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:133)
at org.testng.internal.TestInvoker.invokeMethod(TestInvoker.java:598)
at org.testng.internal.TestInvoker.invokeTestMethod(TestInvoker.java:173)
at org.testng.internal.MethodRunner.runInSequence(MethodRunner.java:46)
at org.testng.internal.TestInvoker$MethodInvocationAgent.invoke(TestInvoker.java:824)
at org.testng.internal.TestInvoker.invokeTestMethods(TestInvoker.java:146)
at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:146)
at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:128)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1596)
at org.testng.TestRunner.privateRun(TestRunner.java:794)
at org.testng.TestRunner.run(TestRunner.java:596)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:377)
at org.testng.SuiteRunner.access$000(SuiteRunner.java:28)
at org.testng.SuiteRunner$SuiteWorker.run(SuiteRunner.java:418)
at org.testng.internal.thread.ThreadUtil.lambda$execute$0(ThreadUtil.java:64)
at java.base/java.util.concurrent.FutureTask.run(FutureTask.java:317)
at java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1144)
at java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:642)
at java.base/java.lang.Thread.run(Thread.java:1583)
]]>
    </failure>
  </testcase> <!-- homePageHeadersTest -->
  <system-out/>
  <testcase classname="com.qa.crm.tests.HomePageTest" name="searchCountTest" time="1.157"/>
  <system-out/>
  <testcase classname="com.qa.crm.tests.HomePageTest" name="searchCountTest" time="1.154"/>
  <system-out/>
  <testcase classname="com.qa.crm.tests.HomePageTest" name="searchCountTest" time="1.357"/>
  <system-out/>
  <testcase classname="com.qa.crm.tests.HomePageTest" name="returnBackToHomePageTest" time="0.071"/>
  <system-out/>
  <testcase classname="com.qa.crm.tests.HomePageTest" name="searchTest" time="2.327"/>
  <system-out/>
  <testcase classname="com.qa.crm.tests.HomePageTest" name="searchTest" time="1.916"/>
  <system-out/>
  <testcase classname="com.qa.crm.tests.HomePageTest" name="searchTest" time="1.911"/>
  <system-out/>
</testsuite> <!-- com.qa.crm.tests.HomePageTest -->
